### ISSUE TITLE
Q: Ancient dependencies in examples/sandbox?

### ISSUE THREAD

**Issue by jbrockmendel** [statsmodels/statsmodels#5752]
There are a number of places where `numdifftools`, `nipy`, or `nitime` are imported or otherwise mentioned.  Are these going to be reinstated as dependencies at some point?  Or can the imports at least be cleared at and replaced with statsmodels-internal versions?

**Comment by josef-pkt** [comment]
> replaced with statsmodels-internal versions

No, that wasn't the point. In those cases I have written similar functions for statsmodels to what is in the other packages and the code was to compare behavior across packages.

Those packages were never required dependencies for statsmodels. Those were for "manual" comparisons in scripts.

**Comment by jbrockmendel** [comment]
> Those packages were never required dependencies for statsmodels. Those were for "manual" comparisons in scripts.

What subset of files does this apply to?  e.g. `try_ld_nitime` (see also #5719) makes sense as a "this file exists specifically to compare behavior with `nitime`".  But it doesn't make as much sense for e.g. `tsa.mlemodel`, `garch`, and `distributions.estimators` which use `numdifftools` directly.  (and `tsa.arma_mle` has a comment incorrectly saying it uses `numdifftools`)

**Comment by josef-pkt** [comment]
Why don't you write some nice documentation notebooks for topics that are still missing instead of dusting the attic?

Distributions was one of my favorite topics and I have additional private code for it, but I gave up when a scipy developer pissed me off. There are a large amount of goodies in distributions that are another part that needs serious work to make it user friendly and ready for prime time. 
Some of the distribution.estimators where in interaction with a scipy contributor that has a package for extreme value theory and application (but the package is GPL AFAIR)

**Comment by jbrockmendel** [comment]
> Why don't you write some nice documentation notebooks for topics that are still missing instead of dusting the attic?

For the same reasons as always.  This part of the conversation is tiresome.

Speaking of docs: https://github.com/statsmodels/statsmodels/blob/5baf2dd92c43135be55ba4970122ed5477a8c355/docs/source/dev/get_involved.rst#sandbox

> [...] Many of these models and functions are close to done, however, and we welcome any and all contributions to complete them, including refactoring, documentation, and tests

> Distributions was one of my favorite topics and I have additional private code for it

Neat!  I look forward to seeing more of it.

To the original question: where did we land on numdifftools?

**Comment by josef-pkt** [comment]
> Speaking of docs: https://github.com/statsmodels/statsmodels/blob/5baf2dd92c43135be55ba4970122ed5477a8c355/docs/source/dev/get_involved.rst#sandbox

this paragraph is outdated and should be deleted

Another reason to work on docs instead of worrying about numdifftools

**Comment by jbrockmendel** [comment]
Related:
- [ ] can references to "scikits.statsmodels.foo" be updated to just "statsmodels.foo"? (only in cases where "statsmodels.foo" still exists)
- [ ] Are references to "scikits.timeseries" still valid?
